plugins/github-deployments/README.md

```diff
@@ -0,0 +1,67 @@
+# GitHub Deployments Plugin
+
+The GitHub Deployments Plugin displays recent deployments from GitHub.
+
+![github-deployments-card](./docs/github-deployments-card.png)
+
+## Getting Started
+
+1. Provide OAuth credentials:
+
+- Create an [OAuth App](https://developer.github.com/apps/building-oauth-apps/creating-an-oauth-app/) and set env variables for ID and Secret.
+
+```bash
+export AUTH_GITHUB_CLIENT_ID={{YOUR_CLIENT_ID}}
+export AUTH_CLIENT_SECRET={{YOUR_CLIENT_SECRET}}
+```
```

Comment:
i think this section could be replaced with just a prerequisites link to https://backstage.io/docs/auth/github/provider ?